build.py

```diff
@@ -409,7 +413,8 @@ def onnxruntime_cmake_args(images, library_paths):
                 cargs.append('-DTRITON_BUILD_CONTAINER_VERSION={}'.format(
                     TRITON_VERSION_MAP[FLAGS.version][1]))
 
-            if TRITON_VERSION_MAP[FLAGS.version][3] is not None:
+            if TRITON_VERSION_MAP[
+                    FLAGS.version][3] is not None and FLAGS.enable_gpu:
```

Comment:
I don't think openvino is GPU based. @tanmayv25 any thoughts?

Reply:
There are openvino plugins for GPU. See [here](https://github.com/openvinotoolkit/openvino/blob/master/docs/IE_PLUGIN_DG/Plugin.md). They even have a plugin wrapping around cuDNN for supporting NVIDIA GPUs. Though the support is marked experimental.

I would confirm whether onnx-openvino support GPU plugins.

Reply:
Anyways, onnx-openvino is supported on CPU. Hence, I would advice adding the `-DTRITON_ENABLE_ONNXRUNTIME_OPENVINO=ON` for cpu only build too.